docs/self-managed/platform-deployment/helm-kubernetes/platforms/amazon-eks/dual-region.md

```diff
@@ -0,0 +1,559 @@
+```
+
+```bash
+kubectl --context $CLUSTER_1 apply -f https://github.com/camunda/c8-multi-region/blob/main/test/resources/aws/2-region/kubernetes/nginx.yml -n $CAMUNDA_NAMESPACE_1
+kubectl --context $CLUSTER_2 apply -f https://github.com/camunda/c8-multi-region/blob/main/test/resources/aws/2-region/kubernetes/nginx.yml -n $CAMUNDA_NAMESPACE_2
+```
+
+2. Try to curl each other from within the other pod.
+
+```bash
+kubectl --context $CLUSTER_1 exec -n $CAMUNDA_NAMESPACE_1 -it sample-nginx -- curl http://sample-nginx.sample-nginx-peer.camunda-paris.svc.cluster.local
+
+kubectl --context $CLUSTER_2 exec -n $CAMUNDA_NAMESPACE_2 -it sample-nginx -- curl http://sample-nginx.sample-nginx-peer.camunda-london.svc.cluster.local
+
+# http://sample-nginx.sample-nginx-peer.camunda-london.svc.cluster.local
+# http://$pod.$service.$namespace.svc.cluster.local
+```
+
+3. After a successful retrieval of the remote nginx start page, you can remove this temporary setup.
+
+```bash
+kubectl --context $CLUSTER_1 delete -f https://github.com/camunda/c8-multi-region/blob/main/test/resources/aws/2-region/kubernetes/nginx.yml -n $CAMUNDA_NAMESPACE_1
+kubectl --context $CLUSTER_2 delete -f https://github.com/camunda/c8-multi-region/blob/main/test/resources/aws/2-region/kubernetes/nginx.yml -n $CAMUNDA_NAMESPACE_2
+```
+
+</TabItem>
+</Tabs>
+
+## Deploy Camunda 8 to the Clusters
+
+<Tabs queryString="deploy">
+<TabItem value="elastic-secret" label="Create the secret for ElasticSearch">
+
+ElasticSearch will use S3 as a backup and restore bucket and for that, you need to configure a Kubernetes secret to not expose those in cleartext.
+
+You can pull the data from Terraform since you exposed those via the `output.tf`.
+
+1. Where your `config.tf` and `output.tf` live, execute the following to export the access keys to environment variables. This will allow an easier creation of the Kubernetes secret via the command line.
+
+```bash
+export ACCESS_KEY=$(terraform output -raw s3_aws_access_key_id)
+export SECRET_ACCESS_KEY=$(terraform output -raw s3_aws_secret_access_key)
+```
+
+2. Create the Kubernetes secret within the `camunda-london` and `camunda-paris` namespace on each respective cluster.
+
+```bash
+kubectl --context $CLUSTER_1 -n $CAMUNDA_NAMESPACE_1 create secret generic elasticsearch-env-secret \
+    --from-literal=S3_ACCESS_KEY=$ACCESS_KEY \
+    --from-literal=S3_SECRET_KEY=$SECRET_ACCESS_KEY
+
+kubectl --context $CLUSTER_2 -n $CAMUNDA_NAMESPACE_2 create secret generic elasticsearch-env-secret \
+    --from-literal=S3_ACCESS_KEY=$ACCESS_KEY \
+    --from-literal=S3_SECRET_KEY=$SECRET_ACCESS_KEY
+```
+
+</TabItem>
+<TabItem value="helm-chart" label="Camunda 8 Helm Chart prerequisites">
+
+Our recommendation is to work with layered helm values files since you will want to have a base `value.yml` that is generally applicable and then two overlays that are for `region0` and `region1`.
```

Comment:
## [big one]

Same suggestion as for the TF files - does it make sense to instead to the users to `git clone` https://github.com/camunda/c8-multi-region/tree/aws-operational/test/resources/aws/2-region/kubernetes, adjust the scripts accordingly, so we don't need to explain what files with what contents to create?

Reply:
the proposed change is included in the latest commit - https://github.com/camunda/camunda-docs/pull/3430/commits/4e11aa0a397cdf49f8810c07b254f8420fb23e58